lib/project_types/script/commands/enable.rb

```diff
@@ -9,6 +9,7 @@ class Enable < ShopifyCli::Command
       end
 
       def call(args, _name)
+        @ctx.puts(@ctx.message('script.enable.info'))
         form = Forms::Enable.ask(@ctx, args, options.flags)
         return @ctx.puts(self.class.help) unless form
 
```

Comment:
I think the UX requirement wasn't perfectly clear.  In my head, I kind of assumed that this tip on `enable.info` would be displayed after the command is executed successfully, and not at the very beginning.  Unless @abater1970 has decided otherwise from CLI existing UX patterns?